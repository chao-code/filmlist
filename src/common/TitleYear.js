import PropTypes from 'prop-types'

const TitleYear = ({ title, year }) => (
  <span>
    {title} <small className="text-muted">({year})</small>
  </span>
)

TitleYear.propTypes = {
  title: PropTypes.string.isRequired,
  year: PropTypes.string.isRequired,
}

export default TitleYear
