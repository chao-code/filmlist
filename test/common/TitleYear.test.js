import { shallow } from 'enzyme'
import TitleYear from '../../src/common/TitleYear'

describe('<TitleYear />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<TitleYear title="title" year="year" />)
    expect(wrapper).toMatchSnapshot()
  })
})
