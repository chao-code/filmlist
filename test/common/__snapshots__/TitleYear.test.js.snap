// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleYear /> should render correctly 1`] = `
<span>
  title
   
  <small
    className="text-muted"
  >
    (
    year
    )
  </small>
</span>
`;
